pages/blog/release-notes-2.4.0.md

```diff
@@ -0,0 +1,85 @@
+---
+title: AsyncAPI Spec 2.4.0 Release Notes
+date: 2022-04-25T19:00:00+01:00
+type: Communication
+tags:
+  - Specification
+  - Release Notes
+cover: /img/posts/release-notes-2.4.0/cover.webp
+authors:
+  - name: Sergio Moya
+    photo: /img/avatars/smoya.webp
+    link: https://twitter.com/smoyac
+    byline: Pilot at AsyncAPI Airlines
+excerpt: 'AsyncAPI 2.4 is now released. This brings ... TBD ...'
+featured: true
+---
+
+The new version of the AsyncAPI specification - 2.4.0 - is now available.
+
+> This is a minor release, and it doesn't bring any breaking changes. You can switch to it by modifying the following value in your AsyncAPI file `asyncapi: '2.3.0'` into `asyncapi: '2.4.0'`
+
+## Server Variables can be now referenced from components
```

Comment:
@danielkocot would you like to review the release notes I wrote here about your recent contribution? Thanks!

Reply:
Review done. The example looks right.

Reply:
Thanks!